lib/memfs/dir.rb

```diff
@@ -17,6 +17,12 @@ def self.chdir(path, &block)
       0
     end
 
+    if Gem::Requirement.new('>= 2.6').satisfied_by?(Gem::Version.new(RUBY_VERSION))
```

Comment:
We will probably see this pattern appear more and more in the code base, it could be a good thing to abstract it in some way 🤔

Something in the idea of

```ruby
if ruby_version_gte?('2.6')
  def xxx
  end
end
```

What do you think?

Reply:
Since it's also used in the specs maybe something like `Memfs.ruby_version_gte?` 🤷‍♂  I'm open to any proposition ^^